src/pages/GamePage.js

```diff
@@ -0,0 +1,19 @@
+import { useParams } from 'react-router-dom';
+import 'App.css';
+import 'style.css';

```

Comment:
Do you need these in all files? If you want to scope the CSS for each component you could take a look at [CSS modules](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/#buttonmodulecss) Since you are using react it's as easy as 
```javascript
// styles.module.css
.sidenav {
  height: 100%;
  min-width: 300px;
  width: 300px;
  position: fixed;
  z-index: 1;
  top: 0;
  left: 0;
  background-color: #17171b;
  overflow-x: hidden;
  padding-top: 20px;
  border-right-style: solid;
  border-right-width: 4px;
  border-right-color: #1e1e21;
}

// Component.jsx
import styles from 'style.module.css'

const component = () => {
    return (
       <div className={styles.sidenav}>
        <h4 className=styles.sidenavTitle>Series</h4>
      </div>
    );
}
```